owslib/util.py

```diff
@@ -170,14 +170,14 @@ def openURL(url_base, data=None, method='Get', cookies=None, username=None, pass
             auth.password = password
         if cert:
             auth.cert = cert
-        if verify and not auth.verify:
-            auth.verify = verify
+        if not verify or not auth.verify:
+            verify = False
```

Comment:
For me, this look strange, it can be simplified to 
```suggestion
        verify = verify and auth.verify
```

Reply:
right @sbrunner!